roles/rsyslog/defaults/main.yaml

```diff
@@ -301,6 +292,22 @@ rsyslog_weight_map:
   'input': '90'
   'inputs': '90'
 
+rsyslog_severity_to_level:
```

Comment:
You don't need to include dictionary keys in quotes/apostrophes.

Concerning the values. Are they really supposed to be strings (enclosed in apostrophes) rather than integers? Wouldn't integers be more natural? (Similar question applies to the already existing `rsyslog_weight_map` above.)

Also, is this variable supposed to be user-settable? It does not seem so, it seems rather internal. If so, it should not be in defaults but in vars and prefixed with underscores (as an internal variable).

Reply:
Ahhh, thank you for finding it out, @pcahyna!  I introduced it at one time, but it turned out not needed.  I'll eliminate rsyslog_severity_to_level.  (and when I find it's necessary again, I'll  use integer... and starting with '_')  Thanks!

Reply:
@pcahyna, BTW, this is also internal: rsyslog_weight_map.  And I believe there are more of them.  Do you think I should change the internal name as you suggested?  Is it ok to update https://github.com/linux-system-roles/logging/pull/85 to take care of them?  Or do you prefer to have it in another new PR?

Reply:
You can do this cleanup in #85 or another PR. Please don't try to fix such stylistic issues here to keep the PR focused, just avoid introducing new ones.

Reply:
To answer the original question, another new PR would be preferred, so you can merge #85 faster and avoid conflicts.

Reply:
> To answer the original question, another new PR would be preferred, so you can merge #85 faster and avoid conflicts.

Eliminated `rsyslog_severity_to_level`.  Thanks!